packages/client/src/pages/LeaderBoard/elements/TableCell/type.ts

```diff
@@ -0,0 +1,7 @@
+export type TableCellProps = {
+  content: React.ReactNode
```

Comment:
Здесь не хватает импорта реакта, но лучше вообще так 
```suggestion
  content: ReactNode
```